packages/aragon-wrapper/src/utils/index.js

```diff
@@ -51,20 +49,13 @@ export function makeProxyFromABI (address, abi, web3, initializationBlock) {
   return new ContractProxy(address, abi, web3, initializationBlock)
 }
 
-export async function getRecommendedGasLimit (web3, estimatedGasLimit, { gasFuzzFactor = DEFAULT_GAS_FUZZ_FACTOR } = {}) {
-  const latestBlock = await web3.eth.getBlock('latest')
-  const latestBlockGasLimit = latestBlock.gasLimit
-
-  const upperGasLimit = Math.round(latestBlockGasLimit * PREVIOUS_BLOCK_GAS_LIMIT_FACTOR)
-  const bufferedGasLimit = Math.round(estimatedGasLimit * gasFuzzFactor)
-
-  if (estimatedGasLimit > upperGasLimit) {
-    // TODO: Consider whether we should throw an error rather than returning with a high gas limit
-    return estimatedGasLimit
-  } else if (bufferedGasLimit < upperGasLimit) {
-    return bufferedGasLimit
-  } else {
-    return upperGasLimit
+export function findMethodOnAppFromData (data, app) {
```

Comment:
What type does data have? Can this function be documented?

Reply:
`data` is the calldata ([`function selector`](https://solidity.readthedocs.io/en/v0.5.7/abi-spec.html#function-selector) + encoded arguments)